stage0/gc.go

```diff
@@ -87,10 +94,11 @@ func getMountsForPrefix(path string, mi io.Reader) (mounts, error) {
 			parentId   int
 			discard    string
 			mountPoint string
+			mode       string
 		)
 
-		_, err := fmt.Sscanf(sc.Text(), "%d %d %s %s %s",
-			&mountId, &parentId, &discard, &discard, &mountPoint)
+		_, err := fmt.Sscanf(sc.Text(), "%d %d %s %s %s %s %s",
+			&mountId, &parentId, &discard, &discard, &mountPoint, &discard, &mode)
```

Comment:
The mode is more difficult to parse than that: according to `man proc`:
```
       /proc/[pid]/mountinfo (since Linux 2.6.26)
              This file contains information about mount points.  It contains lines of the form:

              36 35 98:0 /mnt1 /mnt2 rw,noatime master:1 - ext3 /dev/root rw,errors=continue
              (1)(2)(3)   (4)   (5)      (6)      (7)   (8) (9)   (10)         (11)

              (7)  optional fields: zero or more fields of the form "tag[:value]".
```

I tested a scenario where I have:
```
102 85 0:27 / /tmp rw shared:65 master:16 - tmpfs tmpfs rw
```
So, I have 2 optional field: shared and master.